Database/Corrections/cataObjectFixes.lua

```diff
@@ -333,6 +333,11 @@ function CataObjectFixes.Load()
             [objectKeys.spawns] = {[zoneIDs.FELWOOD]={{49.03,84.95},{48.74,84.98},{48.58,84.43},{48.34,84.62},{46.92,88.14},{46.52,90.53},{45.78,86.46},{47.34,85.70},{48.19,87.39},{51.19,84.19},{50.76,85.02},{49.94,86.24},{49.74,86.48},{49.12,86.24},{48.73,86.30},{48.75,87.55},{48.83,88.62},{49.46,89.44},{48.89,89.28},{48.29,89.76},{49.08,90.81},{49.00,91.35},{48.47,92.23},{48.23,92.26},{47.74,91.68},{48.35,91.20},{48.35,91.20}}},
             [objectKeys.zoneID] = zoneIDs.FELWOOD,
         },
+        [460016] = {
+            [objectKeys.name] = "Felwood Rich Soil",
```

Comment:
Same as before: The name needs to match.

Reply:
Yet another visual marker that is not a 'thing'. Would it be better to set the coords as drop locations for the quest bunny npc instead? (Irontree Soil Credit)

Reply:
Nah, it's good to replace the bunny objectives.